### ISSUE TITLE
Memory Error when training the model

### ISSUE THREAD

**Issue by mustaeenqazi** [deepak112/Keras-SRGAN#35]
Hi Deepak, Thanks for the amazing code, a small issue, when i start training the model with your simplified version of code, the Ram gets exhausted. although i have 12GB ram on my google colab, I tried reducing the size of images but no results

**Comment by hongyanghu** [comment]
have you solve the problem?pls

**Comment by rcbrigg** [comment]
I had the same problem. What batch size are you using? Reducing that should help. I also halved the number of neurons in the dense layers and number of filters in the conv layers, which of course will impact performance.

**Comment by mustaeenqazi** [comment]
The problem is solved by using small batch size and lower number of images. but you can also try to upgrade your memory.

**Comment by BassantTolba1234** [comment]
Dear Sir,
Brilliant work , thank you for sharing your code..
Please, according to your GPU version what the time did you take to finish training your network ??
I'm waiting for your fast reply..

**Comment by BassantTolba1234** [comment]
Dear Sir,
Brilliant work , thank you for sharing your code..
Please, according to your GPU version what the time did you take to finish training your network ??
I'm waiting for your fast reply..

**Comment by BassantTolba1234** [comment]
Please all,
I need the code of implementation this part 
the part is 

{The SRResNet networks
were trained with a learning rate of 10−4 and 106 update
iterations. We employed the trained MSE-based SRResNet
network as initialization for the generator when training
the actual GAN to avoid undesired local optima.{

**Comment by MuterDear** [comment]
Could you please tell me which size of your image before super resolution?
